torch_xla_py/xla_model.py

```diff
@@ -237,7 +238,7 @@ def xla_loss(loss_fn, output_xla_tensors, labels):
             flat_tensors[flat_index].requires_grad = True
             replica_outputs.append(flat_tensors[flat_index])
             flat_index += 1
-        losses.append(loss_fn(*replica_outputs, labels[i]))
+        losses.append(loss_fn(tuple(replica_outputs), labels[i]))
```

Comment:
Hmm, ... the loss function does not take a tuple.

Reply:
Ok, how to unpack it then?

Reply:
Leave as is?

Reply:
Also, I think this function is actually dead, let me remove it.

Reply:
No please, leave it. We might need it for API extensions ...

Reply:
I cannot leave it "as is", this is a syntax error in Python 2. So, what to do about it, especially without test coverage?

Reply:
That should work!

Reply:
Nope:

```
  File "build/bdist.linux-x86_64/egg/torch_xla_py/xla_model.py", line 241
    losses.append(loss_fn((*(list(replica_outputs).append(labels[i])))))
                           ^
SyntaxError: invalid syntax
```

Reply:
That worked for me ...

```
Python 2.7.13 (default, Nov 24 2017, 17:33:09) 
[GCC 6.3.0 20170516] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> def f(a, b):
...   print(a,b)
... 
>>> f(*(list([1,2])))
(1, 2)
>>> 
```

Reply:
The same snippet works for me as well, but for some reason the Python parser chokes for the change we actually need.

Reply:
Also, doesn't list append return `None`?

Reply:
my bad, I didn't check the return type of list append